configuration.md

```diff
@@ -18,23 +18,32 @@ This meaning of each flag is transcendant to the correspondant options in the co
 - ```-p, --port``` is the port to listen to. Defaults 0 (random free port).
 - ```--staticgen``` specifies if you want to enable a Static Website Generator (jekyll and hugo are available).
 - ```-v, --version``` prints the version of the executable.
-- ```--alternative-recaptcha``` replaces `https://www.google.com` to `https://recaptcha.net` in ReCaptcha handling and serving, especially useful in China. See [gh-filebrowser#366](https://github.com/filebrowser/filebrowser/issues/366) for details. Defaults to `false`.
-- ```--recaptcha-key``` is the ReCAPTCHA site key. Enables ReCAPTCHA on login.
-- ```--recaptcha-secret``` is the ReCAPTCHA secret key. Enables ReCAPTCHA on login.
+
+ReCAPTCHA specific options:
+
+- ```--recaptcha.host``` is the ReCAPTCHA site host. By default we use Google's, but some users in China might find it more useful to use `https://recaptcha.net`.
+- ```--recaptcha.key``` is the ReCAPTCHA site key. Enables ReCAPTCHA on login.
+- ```--recaptcha.secret``` is the ReCAPTCHA secret key. Enables ReCAPTCHA on login.
+
+Authentication specific options:
+
+- ```--auth.method``` it the method of authentication between 'none', 'default' and 'proxy' authentication. Defaults to "default".
+- ```--auth.header``` is the header name used for proxy authentication. Default to "X-Forwarded-User".
```

Comment:
Any other reference/explanation is required related to filebrowser/filebrowser#485?